src/hooks/useVenueId.ts

```diff
@@ -11,16 +11,18 @@ const venuePaths = [
   "/v/:venueId",
   "/admin_v2/venue/:venueId",
 ];
-export const useVenueId: () => string | undefined = () => {
+export const useVenueId: () => string | null = () => {
   const history = useHistory();
 
   for (const path of venuePaths) {
     const match = matchPath<VenueRoute>(history.location.pathname, {
       path,
     });
 
-    if (match && match.params.venueId) {
+    if (match?.params.venueId) {
       return match.params.venueId;
     }
   }
+
+  return null;
```

Comment:
I still don't understand what about using `null` here makes this 'better' or 'more right' than using `undefined`?